### ISSUE TITLE
ToggleSwitch - fix accessibility violations

### ISSUE THREAD

**Issue by thrbnhrtmnn** [deven-org/B01LER-Kitchen#778]
#### Description / User Story

The following errors and incomplete tests are thrown by the storybook accessibility addon and should be fixed:
- Incomplete: Form field must not have multiple label elements (all states & variants)

----
#### Requirements
- [x] #542 

----
#### Acceptance Criteria
- [ ] All errors and incomplete tests thrown by the accessibility addon are resolved:
   - [ ] Incomplete: Form field must not have multiple label elements (all states & variants)

----
#### Background information
- ...
